shell/common/rasterizer.cc

```diff
@@ -143,7 +148,10 @@ static sk_sp<SkPicture> ScreenshotLayerTreeAsPicture(
 
   frame->Raster(*tree, true);
 
-  return recorder.finishRecordingAsPicture();
+  SkSerialProcs procs;
```

Comment:
Please zero out your structs. Just in case Skia adds a member without a default initialization.